### ISSUE TITLE
Installing  @typescript-eslint/eslint-plugin fails with 404

### ISSUE THREAD

**Issue by djschilling** [typescript-eslint/typescript-eslint#2363]
When running `npm install` i am getting the following error:

```
npm ERR! prepareGitDep npm ERR! code E404
npm ERR! prepareGitDep npm ERR! 404 Not Found - GET https://registry.npmjs.org/@typescript-eslint%2feslint-plugin - Not found
npm ERR! prepareGitDep npm ERR! 404
npm ERR! prepareGitDep npm ERR! 404  '@typescript-eslint/eslint-plugin@^1.11.0' is not in the npm registry.
npm ERR! prepareGitDep npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! prepareGitDep npm ERR! 404 It was specified as a dependency of 'git-clone-6902e3b8'
npm ERR! prepareGitDep npm ERR! 404
npm ERR! prepareGitDep npm ERR! 404 Note that you can also install from a
npm ERR! prepareGitDep npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! prepareGitDep
npm ERR! prepareGitDep npm ERR! A complete log of this run can be found in:
npm ERR! prepareGitDep npm ERR!     /Users/david/.npm/_logs/2020-08-06T10_26_51_684Z-debug.log
npm ERR! prepareGitDep
```

30 minutes ago it was working.

**Comment by abbasogaji** [comment]
@djschilling i noticed this bug too, its currently stopping my deployment

**Comment by abbasogaji** [comment]
@djschilling hopefully this is a quick fix

**Comment by nrail-joerg-walter** [comment]
`error An unexpected error occurred: "https://registry.yarnpkg.com/@typescript-eslint%2feslint-plugin: Not found".`

**Comment by akoshevoi** [comment]
I have a similar problem.
Running create-react-app gives this error

Creating a new React app in F:\Learning\React\project-1.

Installing packages. This might take a couple of minutes.
Installing react, react-dom, and react-scripts with cra-template...

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@typescript-eslint%2fparser - Not found
npm ERR! 404
npm ERR! 404  '@typescript-eslint/parser@^2.10.0' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'react-scripts'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Антон\AppData\Roaming\npm-cache\_logs\2020-08-06T10_32_48_585Z-debug.log

Aborting installation.
  npm install --save --save-exact --loglevel error react react-dom react-scripts cra-template has failed.

Deleting generated file... package.json
Deleting project-1/ from F:\Learning\React
Done.

**Comment by quanghungit** [comment]
+1

**Comment by FellonOne** [comment]
https://status.npmjs.org/ - see that, Package installation - Partial Outage

**Comment by NicoAiko** [comment]
#2361

**Comment by angel-luis** [comment]
How we can avoid this on build? Because it's stopping the whole deployment...

**Comment by Mevlutyildirim** [comment]
+1

**Comment by gltumakov** [comment]
+1

**Comment by akononenko2375** [comment]
+1

**Comment by EdKhachatryan** [comment]
+1

**Comment by mariagus** [comment]
same problem here!

**Comment by menasSawwaf** [comment]
+1 same problem

**Comment by francochau** [comment]
What happened? When is it usually going to takes for the server fix this?

**Comment by SyntaxError843** [comment]
bruh moment

**Comment by Yurasek** [comment]
+1, coffee time

**Comment by zeljkoX** [comment]
+1

**Comment by modecry** [comment]
+

**Comment by modecry** [comment]
+1

**Comment by MaximeSarrato** [comment]
It works now!

**Comment by StefanBurscher** [comment]
There is a problem with NPM installing scoped packages. They are investigating it. (https://status.npmjs.org/)

**Comment by pgksunilkumar** [comment]
error An unexpected error occurred: "https://registry.yarnpkg.com/@typescript-eslint/experimental-utils/-/experimental-utils-3.4.0.tgz: Request failed \"404 Not Found\"".

its currently stopping my development work

**Comment by djschilling** [comment]
@pgksunilkumar thanks for the answer it is working again.

**Comment by jaganbisoyi** [comment]
I also got the same issue today :
 error 404 Not Found - GET https://registry.npmjs.org/@typescript-eslint%2fparser - Not found
 error 404
 error 404 '@typescript-eslint/parser@^2.2.0' is not in the npm registry.
 error 404 You should bug the author to publish it (or use the name yourself!)
 error 404 It was specified as a dependency of 'react-scripts'
 error 404 Note that you can also install from a
 error 404 tarball, folder, http url, or git url.
